src/ui/button/MainButton/MainButton.scss

```
@@ -0,0 +1,9 @@
+@use "../../../styles/utils/variables.scss" as *;
+
+.main--button {
+  background: $violet;
+}
+
+.main--button:hover {
+  background: $light-violet;
+}
```

Вчера говорили также, что это неправильное название
```suggestion
.main--button {
  background: $violet;

  &:hover {
    background: $light-violet;
  }
}

```